package.json

```diff
@@ -74,24 +74,16 @@
 	},
 	"husky": {
 		"hooks": {
-			"pre-commit": "node tools/git-hooks/pre-commit-hook.js",
-			"prepare-commit-msg": "node tools/git-hooks/prepare-commit-msg.js",
-			"post-checkout": "./tools/git-hooks/post-merge-checkout-hook.sh",
-			"post-merge": "./tools/git-hooks/post-merge-checkout-hook.sh"
+			"pre-commit": "node tools/js-tools/git-hooks/pre-commit-hook.js",
+			"prepare-commit-msg": "node tools/js-tools/git-hooks/prepare-commit-msg.js",
+			"post-checkout": "./tools/js-tools/git-hooks/post-merge-checkout-hook.sh",
+			"post-merge": "./tools/js-tools/git-hooks/post-merge-checkout-hook.sh"
 		}
 	},
-	"dependencies": {
-		"chalk": "4.1.1"
-	},
```

Comment:
I notice we're moving this from a dependency to a dev dependency of the tools. Should we keep it around for `tools/check-renovate-ignore-list.js`?

Reply:
I'm not sure that it makes a difference, do we ever not install the JS dev dependencies? And if we don't, would that apply to some of the other deps here too?

Reply:
That's true, it's not really important for our monorepo tools. 👍